### PR TITLE
Feature/addmeds

### LINKED ISSUE #23: Implement `addmeds` functionality
Our patient manager needs to have functionality to add medication for patients.

### Subtasks
- [x] Add functionality for `addmeds` function
- [x] Add documentation for `addmeds` functionality in the User Guide
- [x] Add tests for the new `addmeds` functionality

### PR DESCRIPTION
**Pull request by snajef**
Resolves #23 

Ready for review.

### CODE REVIEW

**Comment by yuntongzhang** [comment]
Looks like this PR contains changes for both `NRIC` (global-impact changes) and `Medication` (feature-based changes). Can you separate the changes into 2 PRs? Also please add keyword `resolves #{issue_number}` in the PR description to link it with existing issue and add `projects` and `milestone` tag for the PR.

File: src/main/java/seedu/address/model/person/Nric.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.person;
```
Comment (yuntongzhang) [inline src/main/java/seedu/address/model/person/Nric.java]: Is this class supposed to be in this PR? I think this have been merged from the NRIC PR.

File: src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <addressbook>
     <!-- Person with invalid name field -->
     <persons>
+    	<nric>S1234567A</nric>
```
Comment (yuntongzhang) [inline src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml]: This line is not aligned. Same for a few more lines below.

File: src/main/java/seedu/address/model/person/Nric.java (hunk 1 above)
Comment (snajef) [reply src/main/java/seedu/address/model/person/Nric.java]: Kinda confused as to why this showed up as a change when it should have been merged from the NRIC PR, not sure tbh.

File: src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml (hunk 2 above)
Comment (snajef) [reply src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml]: Tabs vs spaces, I'll fix this in a commit.

**Comment by snajef** [comment]
Closing the PR due to week 7 LO requirement of forking workflow.

**Pull request closed without merging**